dashboard/config/initializers/devise.rb

```diff
@@ -354,6 +356,16 @@
 
   OmniAuth.config.before_request_phase do |env|
     request = Rack::Request.new(env)
+
+    # To make an OAuth callback accessible, it must be added to the whitelist of each SSO provider.
+    # Rather than repeating this process for every new Global Edition region,
+    # it is more efficient to strip the Global Edition prefix and handle the request as a standard route.
+    # See: +Rack::GlobalEdition+
+    if request.ge_region && request.script_name
+      ge_path_prefix = Cdo::GlobalEdition.path(request.ge_region)
+      request.script_name = request.script_name.sub(ge_path_prefix, '').presence
```

Comment:
Removes the Global Edition path prefix (e.g. `/global/fa`) from OAuth requests set in the middleware: https://github.com/code-dot-org/code-dot-org/blob/c6edd95da283907df4ff6c5c38343fcdfc756d4a/lib/cdo/rack/global_edition.rb#L55-L68